### ISSUE TITLE
map caching

### ISSUE THREAD

**Issue by pavelkokush** [ssaring/sportstracker#195]
Hi, as I see map caching does not supported at the moment, correct? 
I tried to use https://github.com/MazeMap/Leaflet.TileLayer.PouchDBCached, but it does not work for some reason. 
Have you tried any cache solutions?

**Comment by ssaring** [comment]
Right, there's currently no caching support. And I've not tried anything in this direction yet. My goal was to keep it as simple as possible from the developer and user view.
